src/main/java/ui/components/NavigableListView.java

```diff
@@ -111,25 +114,41 @@ public void restoreSelection() {
                 selectedIndex = Optional.of(nextIndex);
 
                 // The next item will be considered selected
-                onItemSelected.accept(nextIndex);
+                triggerItemSelected(nextIndex);
             }
         }
     }
 
+    /**
+     * Triggers the selected item on the card at the specified index for actions that are not right clicks
+     */
+    private void triggerItemSelected(int index){
+        onItemSelected.accept(index, !IS_RIGHT_CLICK);
+    }
+
+    /**
+     * Triggers the selected item on the card at the specified {index} for actions that are right clicks
+     */
+    private void triggerItemSelectedSecondary(int index){
```

Comment:
We aren't using the term 'secondary' any more, right? Not sure if we should change this to RightClick.